internal/server/httpServer.go

```diff
@@ -87,10 +87,25 @@ func (s *HTTPServer) CliHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Check for blacklisted commands
+	if containsBlacklistedCommand(strings.ToUpper(diceCmds.Cmd)) {
+		http.Error(w, "Error: One or more commands are not allowed", http.StatusForbidden)
```

Comment:
API can't execute more than 1 command at once.

Reply:
updated the message